html/browser/common.css

```diff
@@ -1,5 +1,5 @@
-body 
-{ 
+body
```

Comment:
keep in mind goonchat doesnt use this .css file

Reply:
yes but the non-nano ui does